Formula/richmd.rb

```diff
@@ -0,0 +1,45 @@
+class Richmd < Formula
+  include Language::Python::Virtualenv
+
+  desc "Format Markdown in the terminal with Rich"
+  homepage "https://github.com/willmcgugan/rich"
+  url "https://files.pythonhosted.org/packages/67/47/765d57cdd54d74d2a70de26d7ffad1109745f14f3c425015905e6749d3b9/rich-9.7.0.tar.gz"
+  sha256 "25583ce533afae71de086ca97cf0aa883a5e1a220dfcc6049a2809a74bc79ca0"
+  license "MIT"
+
+  depends_on "python@3.9"
+
+  resource "colorama" do
+    url "https://files.pythonhosted.org/packages/1f/bb/5d3246097ab77fa083a61bd8d3d527b7ae063c7d8e8671b1cf8c4ec10cbe/colorama-0.4.4.tar.gz"
+    sha256 "5941b2b48a20143d2267e95b1c2a7603ce057ee39fd88e7329b0c292aa16869b"
+  end
+
+  resource "commonmark" do
+    url "https://files.pythonhosted.org/packages/60/48/a60f593447e8f0894ebb7f6e6c1f25dafc5e89c5879fdc9360ae93ff83f0/commonmark-0.9.1.tar.gz"
+    sha256 "452f9dc859be7f06631ddcb328b6919c67984aca654e5fefb3914d54691aed60"
+  end
+
+  resource "Pygments" do
+    url "https://files.pythonhosted.org/packages/29/60/8ff9dcb5eac7f4da327ba9ecb74e1ad783b2d32423c06ef599e48c79b1e1/Pygments-2.7.3.tar.gz"
+    sha256 "ccf3acacf3782cbed4a989426012f1c535c9a90d3a7fc3f16d231b9372d2b716"
+  end
+
+  resource "typing-extensions" do
+    url "https://files.pythonhosted.org/packages/16/06/0f7367eafb692f73158e5c5cbca1aec798cdf78be5167f6415dd4205fa32/typing_extensions-3.7.4.3.tar.gz"
+    sha256 "99d4073b617d30288f569d3f13d2bd7548c3a7e4c8de87db09a9d29bb3a4a60c"
+  end
+
+  def install
+    virtualenv_install_with_resources
+
+    (buildpath/"bin/richmd").write <<~EOS
+      #!/bin/bash
+      #{libexec/"bin/python"} -m rich.markdown $@
+    EOS
+    bin.install "bin/richmd"
```

Comment:
```suggestion
    (buildpath/"richmd").write <<~EOS
      #!/bin/bash
      #{libexec/"bin/python"} -m rich.markdown $@
    EOS
    bin.install "richmd"
```

Reply:
took these changes and also changed `EOS` to `SH` and re-pushed